addon/src/helpers/page-title.ts

```diff
@@ -10,26 +16,28 @@ import { guidFor } from '@ember/object/internals';
  */
 export default class PageTitle extends Helper {
   @service('page-title')
-  tokens;
+  declare tokens: PageTitleService;
 
-  get tokenId() {
+  get tokenId(): string {
     return guidFor(this);
   }
 
   constructor() {
+    // eslint-disable-next-line prefer-rest-params
     super(...arguments);
     this.tokens.push({ id: this.tokenId });
   }
 
-  compute(params, _hash) {
-    let hash = {
+  compute(params: string[], _hash: PageTitleHelperOptions) {
+    const hash: PageTitleToken = {
```

Comment:
This type annotation is not needed. TS will check that you're doing it right when you call the service method!

```suggestion
    const hash = {
```